### ISSUE TITLE
Add Ubuntu 20.04 bootstrap test

### ISSUE THREAD

**Issue by SeanTAllen** [ponylang/ponyup#166]
This is mostly the current "ubuntu" test.

However, it should be renamed to "ubuntu 20.04" and use "ubuntu20.04" as the platform.

Once that is done, a new issue should be opened to add bootstrapping for our general glic builds.
